Header header
bool EStop	# 紧急停车请求 false 0：无请求 true 1：请求制动
uint8 sender	# 3- Exception